lib/payment_processor/clients/braintree/transaction.rb

```diff
@@ -18,11 +18,11 @@ class Transaction < Populator
         # * +:currency+ - Billing currency (required)
         # * +:user+     - Hash of information describing the customer. Must include email, and name (required)
         # * +:customer+ - Instance of existing Braintree customer. Must respond to +customer_id+ (optional)
-        attr_reader :result, :action
+        attr_reader :action, :result
```

Comment:
I see `transact` using `@result` - does `:result` need to be accessible through `attr_reader` any more? I don't see it anywhere.